app/src/main/java/com/pajato/android/gamechat/exp/BaseExperienceFragment.java

```diff
@@ -146,6 +196,8 @@ protected void setupExperience(final Context context, final Dispatcher<ExpFragme
             }
         } else
             // Create a new experience.
-            createExperience(context, dispatcher);
+            if(this instanceof BaseGameExpFragment) {
```

Comment:
While https://source.android.com/source/code-style.html does not explicitly state that spaces should follow keywords it is rare to encounter such style in AOSP code.  I know Bryan likes this style.  I'm not a fan.  I especially don't like the inconsistency so I tend to add spaces in when I'm working on code that does not have the spaces following keywords.